connectors/citrus-openapi/src/main/java/org/citrusframework/openapi/model/v2/Oas20ModelHelper.java

```diff
@@ -157,6 +127,10 @@ public static Map<String, OasSchema> getHeaders(Oas20Response response) {
                 .collect(Collectors.toMap(OasHeader::getName, Oas20ModelHelper::getHeaderSchema));
     }
 
+    private static boolean isFormDataMediaType(String type) {
+        return Arrays.asList("application/x-www-form-urlencoded", "multipart/form-data").contains(type);
```

Comment:
out of curiosity, where did you get these from? because there exist more, I think. at least some `message/*` things.